packages/react-components/react-nav-preview/src/components/NavCategory/index.ts

```diff
@@ -1,5 +1,4 @@
 export { NavCategory } from './NavCategory';
-export type { NavCategorySlots, NavCategoryProps, NavCategoryState } from './NavCategory.types';
+export type { NavCategoryProps, NavCategoryState } from './NavCategory.types';
```

Comment:
we also try to keep the same structure for exports where the top is regular exports and the bottom type exports